Phonebook.Frontend/src/app/shared/components/user/user-detail/user-detail.component.html

```diff
@@ -3,34 +3,25 @@
     <mat-card-header>

```

Comment:
Was hast du in dieser Datei geändert? Ich finde es nicht.

Reply:
nothing. hit save